CHANGES.md

```diff
@@ -1,9 +1,20 @@
-=======================
  CHANGES for crate-pdo
 =======================
 
-Unreleased
-==========
+2018/03/08 1.0.0
+=================
+
+- Added support for SSL via `PDO::CRATE_ATTR_SSL_MODE`
+
+- Updated provisioning using ubuntu xenial 64 and the latest version of crate
+
+### Breaking changes
```

Comment:
Sorry for that: please remove the header and instead add ``BREAKING: `` in front of the next 2 list items. eg.:

- BREAKING: Refactored the...